modules/farm_fd2/src/entrypoints/transplanting/lib.js

```diff
@@ -106,11 +105,13 @@ export async function submitForm(formData) {
       },
       undo: async (results) => {
         if (results['transplantingLog'] != 'undone') {
-          await farmosUtil.deletePlantAsset(results['transplantingAsset'].id);
+          await farmosUtil.deletePlantAsset(
```

Comment:
I don't think that deleting the transplanting log deletes the plant asset in the same way that it does for quantities.  So I think the plant asset needs to be deleted even if the transplanting log has been undone. But you can check that by running the test and seeing if the plant asset is left behind.  If it is then we don't want the `if` statement here.